components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -121,16 +113,6 @@ export const AppRoutes = () => {
         return <LinkedInCallback />;
     }
 
-    // Show new signup flow if:
```

Comment:
Shifted this into `<AppBlockingFlows/>`. We'll be checking and showing it a little earlier in the logic flow, but I believe that will be ok.